docs/integrations/bitbucketCloud/locations.md

```diff
@@ -20,6 +20,11 @@ integrations:
   bitbucketCloud:
     - username: ${BITBUCKET_CLOUD_USERNAME}
       appPassword: ${BITBUCKET_CLOUD_PASSWORD}
+      fetch: # optional
```

Comment:
Might as well remove this nesting level (in the docs below it's not here)

Reply:
This was something I specifically requested because we were introducing a new object that was going to be shared across multiple configurations for customizing fetch. Since I have to add it in another spot, would it not make sense to have the nesting so that there is a consistent indicator across configuration locations, since most likely this throttling will end up in a lot of spots in the future? What if we also want to add more configurations for fetching?